hack/test/replication.storage.openshift.io_volumereplicationclasses.yaml

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Need to update the VolRepClass created for the e2e in minikube-rook-mirror-setup.sh as well to add the schedule parameter.

Reply:
ACK